query-node/mappings/src/content/nft.ts

```diff
@@ -349,13 +349,17 @@ export async function createNft(
     : undefined
   const decodedMetadata = nftIssuanceParameters.nft_metadata.toString()
 
+  // load creator channel
+  const creatorChannel = (await getById(store, Video, video.getId(), ['channel'])).channel
+
   // prepare nft record
   const nft = new OwnedNft({
     id: video.id.toString(),
     video: video,
     ownerMember,
     creatorRoyalty,
     metadata: decodedMetadata,
+    creatorChannel: creatorChannel,
```

Comment:
```suggestion
    creatorChannel: video.channel,
```
There's no need for loading of `creatorChannel` via `getById`. We can rely on `channel` being loaded in `video` (you've ensured that with your change on lane `527`).